chapters/hpmor-chapter-066.tex

```diff
@@ -1,6 +1,6 @@
 \partchapter{Self Actualization}{I}
 
-\lettrinepara{H}{\emph{essitation}} \emph{iss alwayss eassy, rarely usseful.}
+\lettrinepara{H}{\parsel{essitation}} \parsel{is alwayss easy, rarely useful.}
```

Comment:
Good spot! Should also be "{esitation}".

Reply:
Also, remove "essitation" from the "LocalWords" spelling exceptions!

Reply:
thanks

Reply:
removed from there too